### ISSUE TITLE
implement LN anchor outputs

### ISSUE THREAD

**Issue by SomberNight** [spesmilo/electrum#6588]
We should implement the [`option_anchor_outputs` feature](https://github.com/lightningnetwork/lightning-rfc/blob/master/09-features.md) for lightning. (https://github.com/lightningnetwork/lightning-rfc/pull/688)

**Comment by chrissv** [comment]
I am a recent "victim" of a commitment transaction having a too-low fee. I force-closed a channel to a now-defunct node, and the last channel activity (apparently a while ago) resulted in an updated commitment transaction being signed with 9.1 sat/vByte. So when that transaction was broadcast as part of the FC, it is languishing in the mempool. Since I don't own either the input or output addresses, I can't RBF or CPFP. If there were a dust-level output to one of my wallet addresses included in the transaction, I could at least CPFP that transaction to get it processed.

I'm not sure what to do now since I don't have any hope of the transaction being confirmed. Very disappointing (and expensive).

**Comment by ecdsa** [comment]
This has been is merged in #9264 . Remaining issues are tracked with their own github issues.

**Comment by chrissv** [comment]
With this new change, if I have a pre-existing lightning channel established with a node, and I perform a lightning transaction, will it set up the anchor transaction automatically (assuming I use the updated electrum client)?

**Comment by SomberNight** [comment]
No, existing channels cannot be "upgraded". There is no spec for that currently in the BOLTs (though there are proposals).
